pandas/tests/io/test_excel.py

```diff
@@ -64,136 +63,93 @@ def setup_method(self, datapath):
 class ReadingTestsBase(SharedItems):
     # This is based on ExcelWriterBase
 
+    @pytest.fixture(autouse=True)
+    def change_to_data_directory(self, datapath):
```

Comment:
i guess we should have a code-checks rule running `pytest --fixtures` and searching for `no docstring available`

Reply:
I'll add a docstring on next iteration. I think adding the lint rule is better for a separate PR unless this is the only place where that occurs

Reply:
> I think adding the lint rule is better for a separate PR unless this is the only place where that occurs

yeah, sorry, it was just a general observation. 256 fixtures currently without docstrings.

Reply:
> I'll add a docstring on next iteration.

just a one-liner is probably fine.